config/default.yaml

```diff
@@ -16,7 +16,7 @@ services:
     systemPassword: 'dummy_password'
     userRole: 'adoption-citizen-user'
   case:
-    url: 'http://ccd-data-store-api-adoption-cos-api-pr-249.service.core-compute-preview.internal'
+    url: 'https://ccd-data-store-api-adoption-cos-api-pr-249.service.core-compute-preview.internal'
```

Comment:
Remove this file chane, sync with R2master

Reply:
> I don't see jenkins file change in this PR, please enable it.

The ccd pact test cases are not fixed so if i enable this change the PR will fail.